com.gitlab.bitseater.meteo.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "com.gitlab.bitseater.meteo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.gitlab.bitseater.meteo",
+    "build-options" : {
+        "cflags" : "-O2  -Wno-deprecated-declarations",
+        "cxxflags" : "-O2",
+        "env" : {
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "/share/vala",
+        "/man",
+        "*.a",
+        "*.la"
+    ],
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--share=network",
+        "--system-talk-name=org.freedesktop.GeoClue2",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules" : [
+        {
+            "name" : "champlain",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/libchamplain/0.12/libchamplain-0.12.16.tar.xz",
+                    "sha256" : "4a7e31cf7889669aebf04631543af64435edd989685159b804911c6005db908d"
+                }
+            ]
+        },
+        {
+            "name" : "geocode-glib",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://ftp.gnome.org/pub/GNOME/sources/geocode-glib/3.24/geocode-glib-3.24.0.tar.xz",
+                    "sha256" : "19c1fef4fd89eb4bfe6decca45ac45a2eca9bb7933be560ce6c172194840c35e"
+                }
+            ]
+        },
+        {
+            "name" : "libdbusmenu",
+            "config-opts" : [
+                "--with-gtk=3",
+                "--disable-gtk-doc",
+                "--disable-doc",
+                "--disable-dumper"
+            ],
+            "build-options" : {
+                "env" : {
```

Comment:
This won't work with GNOME 3.30 runtime, it used to work due to some patches upstream on freedesktop runtime which are not used anymore. You can fix it using make-install-args I guess.

Reply:
Hi @bilelmoussaoui . Are you talking about :
`                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR" : "/app/share/gir-1.0",
                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR" : "/app/lib/girepository-1.0"`
lines?

Reply:
Yep, like this: https://github.com/flathub/org.gnome.FeedReader/blob/0ec2f910dd97d892fb7818f978cd33d0c7cb5659/org.gnome.FeedReader.json#L31